src/components/PollViewer/PollViewer.vue

```diff
@@ -38,7 +38,7 @@
 					class="results__option">
 					<div class="results__option-title">
 						<p>{{ option }}</p>
-						<p class="percentage">
+						<p v-if="isPollPublic || isPollClosed" class="percentage">
```

Comment:
We need to show it "if results are visible", not because "it is public" - visibility conditions can vary.

Let's move this check to computed, to have it in one place and properly named.